lib/actions-oclif/envs.ts

```diff
@@ -377,7 +377,7 @@ function stringifyVarArray<T = Dictionary<any>>(
 					result[key] = value;
 				}
 			},
-			{},
+			{} as Dictionary<any>,
```

Comment:
This is due to updated lodash types, which means the type of this is preserved, and the default `{}` would fail on the `result[key] = value` assignment